examples/streamgraph.json

```diff
@@ -0,0 +1,18 @@
+{
+  "data": {"url": "data/cars.json"},
+  "mark": "area",
+  "encoding": {
+    "x": {"field": "Year","type": "temporal","timeUnit": "year"},
+    "y": {
+      "field": "Weight_in_lbs",
+      "type": "quantitative",
+      "aggregate": "sum",
+      "axis": {"labels": false,"grid": false,"tickSize": 0}
+    },
+    "color": {"field": "Cylinders","type": "nominal"}
+  },
+  "config": {
+    "mark": {"interpolate": "cardinal"},
+    "stack": {"offset": "center"}
```

Comment:
Actually after thinking more about it, the offset should be `”wiggle”`, so it’s similar to the d3 example: http://bl.ocks.org/mbostock/4060954.